testing/features/step_definitions/navigation_steps.rb

```diff
@@ -0,0 +1,75 @@
+# frozen_string_literal: true
+
+Given("the navigation is on the page") do
+  @component = Navigation::Default.new.tap(&:load)
+end
+
+Given("the navigation is on the page with header links") do 
+  @component = Navigation::WithHeader.new.tap(&:load)
+end
+
+Given("the navigation does not fit on the screen") do 
+  page.driver.browser.manage.window.resize_to(320, 568)
+end
+
+Given("the dropdown menu is already open") do
+  page.driver.browser.manage.window.resize_to(320, 568)
+  click_button("More")
+end
+  
+Then("the dropdown toggle is not present") do 
+  expect(@component).to have_no_button("More")
+end
+
+Then("the dropdown toggle is present") do 
+  expect(@component).to have_button("More")
+end
+
+Then("the dropdown menu is hidden") do
```

Comment:
For this one. Do we want to validate that it has been set to invisible? Or do we not care? (Just noticed you've switched from not present to hidden)

Reply:
Yeah I guess it should be `not present` for consistency 👍